src/App.tsx

```diff
@@ -1,24 +1,216 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, {
+  useContext, useEffect, useMemo, useRef, useState,
+} from 'react';
+import classNames from 'classnames';
+import { Todo } from './types/Todo';
+import {
+  addTodo, deleteTodo, getTodos, updateTodo,
+} from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { ErrorNotification } from
+  './components/ErrorNotification/ErrorNotification';
+import { TodoItem } from './components/Todo';
+import { DispatchContext, StateContext }
+  from './components/TodosContext/TodosContext';
+import { FilterTypes } from './types/FilterTypes';
+import { ActionType } from './types/ActionType';
 
-const USER_ID = 0;
+const USER_ID = 11562;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const state = useContext(StateContext);
+
+  const reducer = useContext(DispatchContext);
+
+  const { todos } = state;
+
+  const [errorMessage, setErrorMessage] = useState('');
+  const [filter, setFilter] = useState('all');
+  const [query, setQuery] = useState('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isDisabled, setIsDisabled] = useState(false);
+
+  const inputRef = useRef<HTMLInputElement | null>(null);
+
+  const filteredTodos = useMemo(() => {
+    switch (filter) {
+      case FilterTypes.active:
+        return todos.filter(todo => !todo.completed);
+
+      case FilterTypes.completed:
+        return todos.filter(todo => todo.completed);
+
+      default:
+        return todos;
+    }
+  }, [todos, filter]);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(response => {
+        reducer({ type: ActionType.GetTodos, payload: response });
+      })
+      .catch(() => {
+        setErrorMessage('Unable to load todos');
+        setTimeout(() => {
+          setErrorMessage('');
+        }, 3000);
+      });
+  }, []);
```

Comment:
Try to fix linter error
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/56752562/cbb61363-3bd3-43b0-a51d-47b6257555d0)